### ISSUE TITLE
How long it take to finish training?

### ISSUE THREAD

**Issue by shartoo** [ksw0306/FloWaveNet#12]
I've trained this project with 10000 sentence speech about 2secs/sentecne ,namely 2.78 hours of training data for about 18hours ,with 43 epoches and 98860 steps now. But the loss decrease to -4.7929 while no clear voice can get in the generated wavs.

**Comment by L0SG** [comment]
For the LJSpeech dataset (~24h audio), we trained the model for a total of ~1000K iterations. This roughly equates to ~1 week using a single NVIDIA TITAN V GPU. Currently, we are focusing only on the LJSpeech dataset so YMMV for the (small) custom dataset.

**Comment by YA07** [comment]
> 

thanks,you almost save my poor computer
